spline-shadow/client-ui/src/app/services/lineage/lineage-graph.service.ts

```diff
@@ -52,10 +55,12 @@ export class LineageGraphService {
     let that = this
```

Comment:
Is it a getter or modifier? The method is called `get...` but in fact it changes state of another observable.

Reply:
yes but it gets the latest selected and convert it. It is a getter that does a conversion if you want.